pkg/kvstore/consul.go

```diff
@@ -407,43 +344,6 @@ func (c *ConsulClient) StartWatch(w *Watcher) {
 	}(w)
 }
 
-// GetWatcher watches for kvstore changes in the given key. Triggers the returned channel
-// every time the key path is changed.
-// FIXME This function is highly tightened to the maxFreeID, change name accordingly
-func (c *ConsulClient) GetWatcher(key string, timeSleep time.Duration) <-chan []policy.NumericIdentity {
-	ch := make(chan []policy.NumericIdentity, 100)
-	go func(ch chan []policy.NumericIdentity) {
-		curSeconds := time.Second
-		var (
-			k   *consulAPI.KVPair
-			q   *consulAPI.QueryMeta
-			qo  consulAPI.QueryOptions
-			err error
-		)
-		for {
-			k, q, err = c.KV().Get(key, &qo)
```

Comment:
Same issue in StartWatch. I can't comment in there as it's not been touched in this PR.